xbmc/music/MusicDatabase.cpp

```diff
@@ -1238,7 +1242,8 @@ bool CMusicDatabase::UpdateSong(CSong& song, bool bArtists /*= true*/, bool bArt
                           song.lastPlayed, //
                           song.rating, song.userrating, song.votes, //
                           song.replayGain, //
-                          song.iBPM, song.iBitRate, song.iSampleRate, song.iChannels);
+                          song.iBPM, song.iBitRate, song.iSampleRate, song.iChannels, //
```

Comment:
What is the trailing `//` good for?

Reply:
No idea tbh.  I just followed what is already there, as this is the way several functions are written in the music database.

Reply:
I'd say if we see no sense in this, let's remove it.

Reply:
I did some digging and its the second answer here https://stackoverflow.com/questions/24974919/how-to-mark-a-region-so-clang-format-wont-touch-it 

Although I've touched this code and the way it ls laid out would make it a prime candidate for clang to reformat, it isn't in the diff that Jenkins produces.  If I select the block of code in my IDE and apply clang-format to the selected block, with the '//'s at the end, nothing happens.  If I remove them, clang re-formats the selected block.

Because it makes the code much more readable, I'd prefer to leave it as the `clang-format on / off` stuff in GUIInfoManager doesn't appear to make a difference.

Reply:
As you which, I will not fight you on this. 

But if everybody invents own coding guidelines, we can stop using clang-format and maintaining a coding guideline. Tricking our tool that ensures that we obey to our (!) own coding guidelines is imo not the way to go. I clang-format is configured that we want to have certain stuff on one line, there is no justification in using tricks like empty comments.